api/pom.xml

```diff
@@ -96,6 +96,11 @@
       </exclusions>
     </dependency>
     <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
That comment probably belonged elsewhere. However, I think that we should not add the YARN dependency to the API. Certainly not for the app id, which is easily converted using the YARN utilities. Other information from YARN: Should such need arise, then it can be made available through the implementation class, no need to pull it into the API.